lib/rspec_controller_helpers/resourceful.rb

```diff
@@ -1,23 +1,43 @@
 RSpec.shared_examples_for 'a resourceful controller' do |model, raw_options|
-  let(:options) { raw_options || {} }
+  let(:options) { ({ endpoint_formats: {} }).merge(raw_options) }
   let(:symbol) { options[:symbol] || model.name.downcase.to_sym }
   let(:factory) { options[:factory] || symbol }
 
-  let(:tests) do
+  let(:endpoints_to_test) do
     all = [:index, :new, :create, :show, :edit, :update, :delete]
     except = options[:except] || []
     options[:only] || (all - except)
```

Comment:
I'd hoist all above to be all_endpoints... this would then just be
`let(:endpoints) { options[:only].present? ? options[:only] : (all_endpoints - except) }`